lib/drivers/medtronic600/medtronic600Driver.js

```diff
@@ -988,6 +991,14 @@ class TransmitPacketRequest extends NGPRequest {
           this.ngpRetries += 1;
         }
         debug(`Multipacket retry: ${this.ngpRetries}`);
```

Comment:
I guess this line can now be removed, as it looks like a duplicate of line 995?

Reply:
Removed the duplicate information from line 995.